### PR TITLE
Map pois v2

### PR DESCRIPTION
**Pull request by derhabdu**
- Any number of sample points and their image pixel locations can be provided
- Program does linear regression on input points
- Requires Scipy now
- Text feature added, see damaged.json for more information.
- Output is very good now.

### CODE REVIEW

**Review by levmckinney — COMMENTED** [review]
Two pretty small things one Its python so you don't need the semicolons and two I'm thinking we should change the key from damaged to POls since we can also use this module to mark areas where things have changed on the map.

File: src/task1/plot_pois/plot_pois.py [hunk 1]
```diff
@@ -5,14 +5,20 @@
...
-_p.add_argument("-ps", '--pinscale', type=float, help="The scale you'd like for the pin image.", default=0.05);
+_p.add_argument("-im", '--inputmap', type=str, help="The location/filename of the input map json file. Default = map.json", default="map.json");
+_p.add_argument('-id', '--inputdamaged', type=str, help="The location/filanem of the damaged panels JSON file. Default = damaged.json", default="damaged.json");
+_p.add_argument("-o", '--output', type=str, help="The output file name. Default = usc_utataero_YYYYMMDD_HHMMSS.jpg", default="usc_utataero_" + dt.now().strftime('%Y%m%d_%H%M%S') + ".jpg");
+_p.add_argument("-pi", '--pinimage', type=str, help="The file name of the marker image. Default = pinpoint.png", default="pinpoint.png");
+_p.add_argument("-ps", '--pinscale', type=float, help="The scale you'd like for the pin image. Default = 0.05", default=0.05);
+_p.add_argument("-fs", '--fontscale', type=float, help="The font size you'd like. Default = 1.0 ", default=1);
+_p.add_argument("-ft", '--fontthickness', type=int, help="The font thickness you'd like. Default = 1 (integers only!)", default=1);
+# TODO: font-positions
```
Comment (Timothylock) [inline src/task1/plot_pois/plot_pois.py:21]: Probably should do that  :)

File: src/task1/plot_pois/plot_pois.py [hunk 2]
```diff
@@ -52,52 +58,70 @@
...
 	print(e);
 	exit();
 
-print('Plot POIs Configuration Setup:');
+print('Plot POIs (v2) Configuration Setup:');
 print('-------------------------');
 print('Map file:', im_file);
 print('Damaged file:', id_file);
 print('Output file:',output_file);
-97
```
Comment (Timothylock) [inline src/task1/plot_pois/plot_pois.py:69]: tyyy

File: src/task1/plot_pois/plot_pois.py [hunk 3]
```diff
@@ -52,52 +58,70 @@
...
 	# at x,y the image is drawn to the right and then bottom
 	# So we go -width/2 and -height
 	pxd = int(px - pin_image_width/2);
 	pxd2 = int(pxd + pin_image_width);
 	pyd = int(py - pin_image_height);
-	pyd2 = int(pyd + pin_image_height);	
+	pyd2 = int(pyd + pin_image_height);		
+	
+	
 	for chan in range(0,3):
```
Comment (Timothylock) [inline src/task1/plot_pois/plot_pois.py:114]: I would see if theres a built in function that can do this all at once... otherwise this is fine

**Review by Timothylock — APPROVED** [review]
Same comments that Lev had and assuming those are fixed, lgtm :shipit:

**Pull request merged**